src/main/java/com/capitalone/dashboard/model/Release.java

```diff
@@ -0,0 +1,85 @@
+package com.capitalone.dashboard.model;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Transient;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import com.capitalone.dashboard.enums.HelmStatus;
+
+/*Model to store the latest release Data*/
+
+@Document(collection="helm_release")
```

Comment:
Is there a way we can work with our current release collection? We feel the data model should not be tool dependent. Let me know what you feel